src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -20,6 +20,8 @@
  */
 class DateCaster
 {
+    const PERIOD_LIMIT = 12;
```

Comment:
why 12? I would have set 3 myself :)
the const should be removed (as we cannot make it private)

Reply:
It's arbitrary but monthly periods can be displayed for a whole year with a limit of 12.

Reply:
3 still looks enough to me, nobody will look at 12 values...